app/src/lib/api.ts

```diff
@@ -374,6 +374,13 @@ export class API {
       return await parsedResponse<IAPIRepository>(response)
     } catch (e) {
       if (e instanceof APIError) {
+        if (org) {
```

Comment:
:art: we're not fans of truthiness here - can you make this a bit more explicit by having a `if (org !== null) {` on this line?